logs/parse_test.go

```diff
@@ -442,6 +442,22 @@ var parseTests = []parseTestpair{
 		},
 		true,
 	},
+	{
+		logs.LogPrefixCustom10,
+		// Database and user are empty
+		"2020-09-04 16:03:11.375 UTC [417880]: [1-1] db=,user= LOG:  pganalyze-collector-identify: myserver",
```

Comment:
This seems to be copied from the example above, but I think log lines like that normally happen in a session context, where a user and database are present, so they would not be empty. Could we use a more realistic log line here? I think copying the line from the test for example 14 (`"2021-11-17 19:06:53.897 UTC [34][autovacuum worker][3/5][22996] LOG:  automatic analyze of table \"mydb.pg_catalog.pg_class\" system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.01 s",`) should work here (you'll of course need to change the prefix to match).

Reply:
Have updated the the test to both actually capture the issue, but also include a realistic log line